src/+otp/+kuramotosivashinsky/f.m

```diff
@@ -0,0 +1,5 @@
+function du = f(~, u, D, L)
+
+du = - L*(L*u + u) - u.*(D*u);
```

Comment:
Might add4 spaces here

Reply:
For the term u u_x, I think it's generally preferable to write as d/dx (u^2) / 2 and discretize as `D u^2 / 2`